sdk/highlight-node/src/index.ts

```diff
@@ -2,3 +2,4 @@ export { H, HIGHLIGHT_REQUEST_HEADER } from './sdk'
 export { Highlight } from './client'
 export { NodeOptions } from './types'
 export * as Handlers from './handlers'
+export { H as ClientH } from 'highlight.run'
```

Comment:
i dont think we want node exporting the client, just `next`. reason for that is the `node` sdk is backend-only, whereas the `next` sdk is full-stack (and applies both to backend nodejs and the browser). sorry for missing that initially!